src/Apps/Order2/Routes/Details/Components/Order2DetailsMessage.tsx

```diff
@@ -82,7 +82,7 @@ const getMessageContent = (order): React.ReactNode => {
         <>
           <Text variant="sm">
             Thank you! Your offer has been submitted. You will receive an email
-            shortly with all the details. Please note making an offer doesn’t
+            shortly with all the details. Please note making an offer doesn't
```

Comment:
I think that we need to keep the curly apostrophe here

Reply:
Oh:( I feel like this is constantly changed every time I save the file :( Need to figure out where is this setting!

Reply:
Its gonna be a stretch to QA this stuff, so I don't think the expectation is that we use curly apostrophes everywhere. That said! need to use a `&rsquo;`. Not scalable tho, being honest.

Reply:
Updated with `&rsquo;` here.